docs/full_stack_server.md

```diff
@@ -98,7 +98,7 @@ services:
         ports:
             - 27017:27017
   keycloak:
-    image: jboss/keycloak:3.4.3.Final
+    image: quay.io/keycloak/keycloak:9.0.3
```

Comment:
Yep.. Actually we working with keycloak 12 at the moment - they are really fast on releasing stuff.

Reply:
I had issues with latest release. I could not login after container was setup. Had to downgrade to 9.0.3. Could be my side or there is really an issue with the latest images.